tests/main/java/tests/Calculator.java

```diff
@@ -0,0 +1,41 @@
+package main.java.tests;
+
+import java.util.Scanner;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        int num1;
+        int num2;
+        int result;
+        char smb;
+        Scanner reader = new Scanner(System.in);
+
+        System.out.print("Enter first num  ");
+        num1 = reader.nextInt();
+
+        System.out.print("Enter smb ");
+        smb = reader.next().charAt(0);
+
+        System.out.print("Enter second num  ");
+        num2 = reader.nextInt();
+
+        switch(smb) {
+            case '+'   : result = num1 + num2;
+                break;
+            case '-'   : result = num1 - num2;
+                break;
+            case '*'   : result = num1 * num2;
+                break;
+            case '/'   : result = num1 / num2;
+                break;
+            default:
+                return;
+
+        }
+        System.out.println("\nResult is :\n" + num1 + " " + smb + " " + num2 + " = " + result);
+
+    }
+
+}
```

Comment:
Dar factorial galėjai pasidaryti nesunkus būtų, 
public static int factorial (int num1, int num2) {
        int num1= 1;
        for (int i = 1; i < num2+1; i++) {
            num1 *= i;
        }
        return factorial;
    } 

tik reiktu atsiminti, kad factorial pirmas numeris būtų visados 1, kad būtų teisingas ats.